### PR TITLE
Adds a new section to the contributing guidelines to explain the procedure for new features and refactoring

### LINKED ISSUE #48: CONTRIBUTING.md has a broken link
#### Summary

The file `CONTRIBUTING.md` has a broken link here:

> If you are working on new features or refactoring, [create a proposal](https://github.com/laminas/.github/blob/main/issues/new?labels=RFC&template=RFC.md&title=%5BRFC%5D%3A+%5BTITLE+OF+RFC%5D).

#### Expected behavior

Where is this supposed to go to?
If to create a RFC against a component, I am not sure how this can be done.
If against this repo, then link can be fixed to https://github.com/laminas/.github/issues/new?labels=RFC&template=RFC.md&title=%5BRFC%5D%3A+%5BTITLE+OF+RFC%5D

Let me know and I will submit a PR accordingly

### PR DESCRIPTION
**Pull request by visto9259**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      |no
| New Feature   | no
| RFC           | no
| QA            | no

### Description
Added a section on new features and refactoring

Fixes #48

### CODE REVIEW

**Comment by visto9259** [comment]
Once we agree on a final text, I will do a PR for Mezzio

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,11 @@ If you wish to contribute to this project, please be sure to read/subscribe to t
 - [Chat](https://laminas.dev/chat)
 - [Code of Conduct](CODE_OF_CONDUCT.md)
 
-If you are working on new features or refactoring, [create a proposal](./issues/new?labels=RFC&template=RFC.md&title=%5BRFC%5D%3A+%5BTITLE+OF+RFC%5D).
+## New Features or Refactoring
+If you are working on new features or refactoring for an existing project item (component, tool, etc.), create a Request For Comments
+(RFC) in the correspondig repository via a new issue report.
+
+If you want to propose a new project item, create a RFC in the [Technical Steering Committee](https://github.com/laminas/technical-steering-committee) repository via a new issue report.
```
Comment (froschdesign) [inline CONTRIBUTING.md]: > …project item…

Can we specify this like above?

Comment (visto9259) [reply CONTRIBUTING.md]: Like this?

> If you want to propose a new project item (component, tool, etc.), create a RFC...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,7 +9,11 @@ If you wish to contribute to this project, please be sure to read/subscribe to t
 - [Chat](https://laminas.dev/chat)
 - [Code of Conduct](CODE_OF_CONDUCT.md)
 
-If you are working on new features or refactoring, [create a proposal](./issues/new?labels=RFC&template=RFC.md&title=%5BRFC%5D%3A+%5BTITLE+OF+RFC%5D).
+## New Features or Refactoring
+If you are working on new features or refactoring for an existing project item (component, tool, etc.), create a Request For Comments
+(RFC) in the correspondig repository via a new issue report.
```
Comment (froschdesign) [inline CONTRIBUTING.md]:
```suggestion
If you are working on new features or refactoring for an existing project item (e.g. adding a new filter or validator or rewriting an existing class), create a Request For Comments (RFC) in the correspondig repository via a new issue report.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (froschdesign) [inline CONTRIBUTING.md]:
```suggestion
If you want to propose a new component/package, create a RFC in the [Technical Steering Committee](https://github.com/laminas/technical-steering-committee) repository via a new issue report.
```

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@visto9259 
Thank you for your time and this contribution! 👍

**Pull request merged**